Classes/Controllers/PBHistorySearchMode.h

```diff
@@ -0,0 +1,25 @@
+//
+//  PBHistorySearchMode.h
+//  GitX
+//
+//  Created by Sven-S. Porst on 2016-12-14
+//
+
+#import <Foundation/NSObjCRuntime.h>
+
+#ifndef PBHistorySearchMode_h
+#define PBHistorySearchMode_h
+
+typedef enum historySearchModes {
```

Comment:
GitX has one other instance of this naming style in `enum branchFilterTypes`, but I think it's better to either name the enum the same as the typedef (`enum PBHistorySearchMode`) or just use an anonymous enum.